meta-lmp-base/recipes-containers/docker-compose/docker-compose_2.4.1.bb

```diff
@@ -20,6 +26,8 @@ GO_EXTRA_LDFLAGS = "-w -X ${GO_IMPORT}/internal.Version=${PV}"
 go_do_compile() {
 	export TMPDIR="${GOTMPDIR}"
 	mkdir -p ${B}/cli-plugins/bin
+	${GO} mod download
```

Comment:
We just need to add back '-modcacherw' here.

Reply:
The clean step will fail later on without it (and failed locally when I used this patch).